packages/api/helper/googleSheetsAPIConnection.js

```diff
@@ -0,0 +1,31 @@
+const { google } = require('googleapis');
+
+const client = new google.auth.JWT(
+  process.env.GOOGLE_SHEETS_CLIENT_EMAIL,
+  null,
+  process.env.GOOGLE_SHEETS_PRIVATE_KEY,
+  ['https://www.googleapis.com/auth/spreadsheets']
+);
+
+client.authorize(function (err, tokens) {
+  if (err) {
+    console.log(err);
+    return;
+  } else {
+    console.log('Connected!');
```

Comment:
If we end up keeping this code, what do you think about making this log line a little more clear, like, "Connected to Google Sheets API"? Same thing with the error log above. That way it'll be easier to understand where these messages are coming from.